html/includes/print-alert-rules.php

```diff
@@ -19,14 +19,18 @@
         'disabled'  => 0,
         'name'      => 'Devices up/down',
     );
-    $default_rules[] = array(
-        'device_id' => '-1',
-        'rule'      => '%devices.uptime < "300" && %macros.device = "1"',
-        'severity'  => 'critical',
-        'extra'     => '{"mute":false,"count":"1","delay":"300"}',
-        'disabled'  => 0,
-        'name'      => 'Device rebooted',
-    );
+
+    if ($config['os'][$device['os']]['bad_uptime'] !== true) {
```

Comment:
This is only in the alert rule section for specific devices. I doubt people will click it from there but imho this isn't the way to deal with this as it will now exclude uptime rules for all devices.